Sources/GryphonLib/Driver.swift

```diff
@@ -192,7 +202,7 @@ public class Driver {
 
 			Compiler.logStart("🧑‍💻  Creating iOS compilation files...")
 
-			try createIOSCompilationFiles(forXcodeProject: xcodeProject, forTarget: target)
+            try createIOSCompilationFiles(forXcodeProject: xcodeProject, useNewBuildSystem: needToUseNewBuildSystem, forTarget: target)
```

Comment:
The new argument probably needs to be considered in the `getRecursiveArguments(from:)` method too, otherwise it won't show up here when we call `gryphon init`.